dev/com.ibm.ws.cdi.extension_fat/fat/src/com/ibm/ws/cdi/extension/apps/invocationContext/MyInterceptor.java

```diff
@@ -0,0 +1,32 @@
+package com.ibm.ws.cdi.extension.apps.invocationContext;
+
+import static org.junit.Assert.assertEquals;
+
+import java.lang.annotation.Annotation;
+import java.util.Set;
+
+import javax.annotation.Priority;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+import com.ibm.ws.cdi.internal.interfaces.CDIUtils;
+
+@Priority(1)
+@Interceptor
+@MyInterceptorBinding
+public class MyInterceptor {
+
+String n = null;
+
+    @AroundInvoke
+    public Object aroundInvokeMethod(InvocationContext ctx) throws Exception {
+        //In a real app this wouldn't be possible, but for now we're just testing this utility method.
+        Set<Annotation> interceptorBindings = CDIUtils.getInterceptorBindingsFromInvocationContext(ctx);
+
+        for (Annotation ann : interceptorBindings) {
+            assertEquals(MyInterceptorBinding.class, ann.annotationType());
+        }
```

Comment:
This should assert that `interceptorBindings` contains the correct set of annotations.

Currently it would pass if `interceptorBindings` is empty.